File: examples/hello.rb
require "cmdstan"

model = CmdStan::Model.new(exe_file: "dist/bin/bernoulli")
data = {"N" => 10, "y" => [0, 1, 0, 0, 0, 0, 0, 0, 0, 1]}
model.sample(data: data, chains: 5)

File: examples/hello.R
library(cmdstanr)

# currently requires CmdStan to be installed
model <- cmdstan_model(exe_file="dist/bin/bernoulli")
data <- list(N=10, y=c(0, 1, 0, 0, 0, 0, 0, 0, 0, 1))
model$sample(data)
